spec/Failsafe.spec.ts

```diff
@@ -54,7 +55,7 @@ describe(`Failsafe`, function() {
         { scripts: [`success`, `success`], expected_result: Success },
         { scripts: [`success`, `general-failure`], expected_result: General_Failure },
         { scripts: [`general-failure`, `general-failure`], expected_result: General_Failure },
-        { scripts: [`general`, `success`], expected_result: General_Failure },
+        { scripts: [`general-failure`, `success`], expected_result: General_Failure },
```

Comment:
I think that was a typo… calling `general` which does not exist, also results in a exit code 1